cmd/syntactic-code-intel-worker/shared/store_helpers_test.go

```diff
@@ -0,0 +1,102 @@
+package shared
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/keegancsmith/sqlf"
+	"github.com/sourcegraph/sourcegraph/internal/database"
+)
+
+func insertIndexRecords(t testing.TB, db database.DB, records ...SyntacticIndexingJob) {
+	for _, index := range records {
+		if index.Commit == "" {
+			index.Commit = makeCommit(index.ID)
+		}
+		if index.State == "" {
+			index.State = "completed"
+		}
+		if index.RepositoryID == 0 {
+			index.RepositoryID = 50
+		}
+		// Ensure we have a repo for the inner join in select queries
+		insertRepo(t, db, index.RepositoryID, index.RepositoryName)
+
+		query := sqlf.Sprintf(`
+			INSERT INTO syntactic_scip_indexing_jobs (
+				id,
+				commit,
+				queued_at,
+				state,
+				failure_message,
+				started_at,
+				finished_at,
+				process_after,
+				num_resets,
+				num_failures,
+				repository_id,
+				should_reindex,
+				enqueuer_user_id
+			) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s, %s)
+		`,
+			index.ID,
+			index.Commit,
+			index.QueuedAt,
+			index.State,
+			index.FailureMessage,
+			index.StartedAt,
+			index.FinishedAt,
+			index.ProcessAfter,
+			index.NumResets,
+			index.NumFailures,
+			index.RepositoryID,
+			index.ShouldReindex,
+			index.EnqueuerUserID,
+		)
+
+		if _, err := db.ExecContext(context.Background(), query.Query(sqlf.PostgresBindVar), query.Args()...); err != nil {
+			t.Fatalf("unexpected error while inserting index: %s", err)
+		}
```

Comment:
```suggestion
		_, err := db.ExecContext(context.Background(), query.Query(sqlf.PostgresBindVar), query.Args()...)
		require.NoError(t, err, "unexpected error while inserting index")
```

Bit more compact. Same could be applied in other places.